src/transit/tests.py

```diff
@@ -6,6 +6,21 @@
 BASE_DEV_URL = environ["TRANSIT_DEV_BACKEND_URL"]
 BASE_PROD_URL = environ["TRANSIT_BACKEND_URL"]
 
+# We want to verify that /allstops returns a list of type 'busStop' only
+def allstops_returns_busStops(r):
```

Comment:
i think we should just do `allstops_returns_busstops` to avoid the camelcase + snakecase craziness

Reply:
I was thinking about that too, but the actual string is `"busStop"`, which is why I wanted to keep it consistent.

Reply:
what about just `allstops_returns_stops` because by def a stop can only be a `busStop`

Reply:
`allstops_returns_bus_stops` imo